src/binary-reader-linker.cc

```diff
@@ -243,11 +261,16 @@ Result BinaryReaderLinker::OnExport(Index index,
                                     ExternalKind kind,
                                     Index item_index,
                                     StringSlice name) {
+  if (kind == ExternalKind::Memory) {
+    WABT_FATAL("Linker does not support exported memories");
+  }
+
```

Comment:
I'm sorry if that comment comes across as discouraging or dismissive of your work.  I do appreciate the work you've been doing that bugs you've been filing.